tests/unit/frameworks/pyramid/http_server_prom_tests.py

```diff
@@ -126,3 +126,106 @@ def test_http_server_metric_collection_method(self, response):
             )
             == expected_response_size_count
         )
+
+    @pytest.mark.parametrize(
```

Comment:
I debated adding a parameterized arg to this test for controlling this rather than wholly duplicating this test. This was easier, but happy to adjust